<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>638</width>
    <height>503</height>
   </rect>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="cursor">
       <cursorShape>ArrowCursor</cursorShape>
      </property>
      <property name="toolTip">
       <string/>
      </property>
      <property name="whatsThis">
       <string/>
      </property>
      <property name="tabShape">
       <enum>QTabWidget::Triangular</enum>
      </property>
      <property name="currentIndex">
       <number>2</number>
      </property>
      <property name="movable">
       <bool>false</bool>
      </property>
      <widget class="QWidget" name="SystemInfo">
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;系统信息&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="whatsThis">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;系统信息&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <attribute name="title">
        <string>系统信息</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QGroupBox" name="groupBox_3">
          <property name="title">
           <string>处理器信息</string>
          </property>
          <widget class="QLabel" name="label_14">
           <property name="geometry">
            <rect>
             <x>40</x>
             <y>30</y>
             <width>67</width>
             <height>17</height>
            </rect>
           </property>
           <property name="text">
            <string>CPU名称:</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_15">
           <property name="geometry">
            <rect>
             <x>40</x>
             <y>60</y>
             <width>67</width>
             <height>17</height>
            </rect>
           </property>
           <property name="text">
            <string>CPU类型:</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_16">
           <property name="geometry">
            <rect>
             <x>40</x>
             <y>90</y>
             <width>67</width>
             <height>17</height>
            </rect>
           </property>
           <property name="text">
            <string>CPU频率:</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_17">
           <property name="geometry">
            <rect>
             <x>40</x>
             <y>120</y>
             <width>81</width>
             <height>17</height>
            </rect>
           </property>
           <property name="text">
            <string>Cache大小:</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_CPUname">
           <property name="geometry">
            <rect>
             <x>130</x>
             <y>30</y>
             <width>421</width>
             <height>17</height>
            </rect>
           </property>
           <property name="text">
            <string>未知</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_CPUtype">
           <property name="geometry">
            <rect>
             <x>130</x>
             <y>60</y>
             <width>431</width>
             <height>17</height>
            </rect>
           </property>
           <property name="text">
            <string>未知</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_CPUfrequency">
           <property name="geometry">
            <rect>
             <x>130</x>
             <y>90</y>
             <width>431</width>
             <height>17</height>
            </rect>
           </property>
           <property name="text">
            <string>未知</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_Cachecapacity">
           <property name="geometry">
            <rect>
             <x>130</x>
             <y>120</y>
             <width>431</width>
             <height>17</height>
            </rect>
           </property>
           <property name="text">
            <string>未知</string>
           </property>
          </widget>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_4">
          <property name="title">
           <string>操作系统信息</string>
          </property>
          <widget class="QLabel" name="label_18">
           <property name="geometry">
            <rect>
             <x>40</x>
             <y>30</y>
             <width>101</width>
             <height>17</height>
            </rect>
           </property>
           <property name="text">
            <string>操作系统类型:</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_19">
           <property name="geometry">
            <rect>
             <x>40</x>
             <y>60</y>
             <width>101</width>
             <height>17</height>
            </rect>
           </property>
           <property name="text">
            <string>操作系统版本:</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_26">
           <property name="geometry">
            <rect>
             <x>40</x>
             <y>90</y>
             <width>67</width>
             <height>17</height>
            </rect>
           </property>
           <property name="text">
            <string>GCC版本:</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_OStype">
           <property name="geometry">
            <rect>
             <x>150</x>
             <y>30</y>
             <width>411</width>
             <height>17</height>
            </rect>
           </property>
           <property name="text">
            <string>未知</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_OSversion">
           <property name="geometry">
            <rect>
             <x>150</x>
             <y>60</y>
             <width>411</width>
             <height>17</height>
            </rect>
           </property>
           <property name="text">
            <string>未知</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_GCCversion">
           <property name="geometry">
            <rect>
             <x>150</x>
             <y>90</y>
             <width>411</width>
             <height>17</height>
            </rect>
           </property>
           <property name="text">
            <string>未知</string>
           </property>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="MemoryInfo">
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;内存信息&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="whatsThis">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;内存信息&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <attribute name="title">
        <string>内存信息</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>80</height>
           </size>
          </property>
          <property name="title">
           <string>CPU</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignJustify|Qt::AlignVCenter</set>
          </property>
          <widget class="QProgressBar" name="progressBar_CPU">
           <property name="geometry">
            <rect>
             <x>50</x>
             <y>30</y>
             <width>431</width>
             <height>23</height>
            </rect>
           </property>
           <property name="value">
            <number>24</number>
           </property>
          </widget>
          <widget class="QLabel" name="label">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>30</y>
             <width>66</width>
             <height>17</height>
            </rect>
           </property>
           <property name="text">
            <string>CPU：</string>
           </property>
          </widget>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_2">
          <property name="title">
           <string>内存和交换分区</string>
          </property>
          <widget class="QProgressBar" name="progressBar_RAM">
           <property name="geometry">
            <rect>
             <x>50</x>
             <y>50</y>
             <width>431</width>
             <height>23</height>
            </rect>
           </property>
           <property name="value">
            <number>24</number>
           </property>
          </widget>
          <widget class="QLabel" name="label_2">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>50</y>
             <width>66</width>
             <height>17</height>
            </rect>
           </property>
           <property name="text">
            <string>内存：</string>
           </property>
          </widget>
          <widget class="QProgressBar" name="progressBar_SWAP">
           <property name="geometry">
            <rect>
             <x>50</x>
             <y>140</y>
             <width>431</width>
             <height>23</height>
            </rect>
           </property>
           <property name="value">
            <number>24</number>
           </property>
          </widget>
          <widget class="QLabel" name="label_3">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>140</y>
             <width>66</width>
             <height>17</height>
            </rect>
           </property>
           <property name="text">
            <string>交换：</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_4">
           <property name="geometry">
            <rect>
             <x>50</x>
             <y>80</y>
             <width>66</width>
             <height>17</height>
            </rect>
           </property>
           <property name="text">
            <string>Used：</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_5">
           <property name="geometry">
            <rect>
             <x>200</x>
             <y>80</y>
             <width>67</width>
             <height>17</height>
            </rect>
           </property>
           <property name="text">
            <string>Left：</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_6">
           <property name="geometry">
            <rect>
             <x>380</x>
             <y>80</y>
             <width>67</width>
             <height>17</height>
            </rect>
           </property>
           <property name="text">
            <string>Total：</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_7">
           <property name="geometry">
            <rect>
             <x>380</x>
             <y>170</y>
             <width>67</width>
             <height>17</height>
            </rect>
           </property>
           <property name="text">
            <string>Total：</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_8">
           <property name="geometry">
            <rect>
             <x>200</x>
             <y>170</y>
             <width>67</width>
             <height>17</height>
            </rect>
           </property>
           <property name="text">
            <string>Left：</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_9">
           <property name="geometry">
            <rect>
             <x>50</x>
             <y>170</y>
             <width>67</width>
             <height>17</height>
            </rect>
           </property>
           <property name="text">
            <string>Used：</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_RAM_USED">
           <property name="geometry">
            <rect>
             <x>100</x>
             <y>80</y>
             <width>67</width>
             <height>17</height>
            </rect>
           </property>
           <property name="text">
            <string>0</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_RAM_LEFT">
           <property name="geometry">
            <rect>
             <x>250</x>
             <y>80</y>
             <width>67</width>
             <height>17</height>
            </rect>
           </property>
           <property name="text">
            <string>0</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_RAM_TOTAL">
           <property name="geometry">
            <rect>
             <x>430</x>
             <y>80</y>
             <width>67</width>
             <height>17</height>
            </rect>
           </property>
           <property name="text">
            <string>0</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_SWAP_USED">
           <property name="geometry">
            <rect>
             <x>100</x>
             <y>170</y>
             <width>67</width>
             <height>17</height>
            </rect>
           </property>
           <property name="text">
            <string>0</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_SWAP_LEFT">
           <property name="geometry">
            <rect>
             <x>250</x>
             <y>170</y>
             <width>67</width>
             <height>17</height>
            </rect>
           </property>
           <property name="text">
            <string>0</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_SWAP_TOTAL">
           <property name="geometry">
            <rect>
             <x>430</x>
             <y>170</y>
             <width>67</width>
             <height>17</height>
            </rect>
           </property>
           <property name="text">
            <string>0</string>
           </property>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="ProcessInfo">
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;进程信息&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="whatsThis">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;进程信息&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <attribute name="title">
        <string>进程信息</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QWidget" name="widget_2" native="true">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QListWidget" name="ProcessInfoList"/>
           </item>
           <item>
            <widget class="QWidget" name="widget_3" native="true">
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <widget class="QPushButton" name="pushButton_refresh">
                <property name="text">
                 <string>刷新</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pushButton_kill">
                <property name="text">
                 <string>结束进程</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="widget_4" native="true">
          <property name="minimumSize">
           <size>
            <width>120</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>120</width>
            <height>16777215</height>
           </size>
          </property>
          <widget class="QLabel" name="label_10">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>10</y>
             <width>67</width>
             <height>17</height>
            </rect>
           </property>
           <property name="text">
            <string>进程数：</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_pNum">
           <property name="geometry">
            <rect>
             <x>70</x>
             <y>10</y>
             <width>67</width>
             <height>17</height>
            </rect>
           </property>
           <property name="text">
            <string>0</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_11">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>40</y>
             <width>67</width>
             <height>17</height>
            </rect>
           </property>
           <property name="text">
            <string>运行数：</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_12">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>70</y>
             <width>67</width>
             <height>17</height>
            </rect>
           </property>
           <property name="text">
            <string>睡眠数：</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_13">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>100</y>
             <width>67</width>
             <height>17</height>
            </rect>
           </property>
           <property name="text">
            <string>僵死数：</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_pRun">
           <property name="geometry">
            <rect>
             <x>70</x>
             <y>40</y>
             <width>67</width>
             <height>17</height>
            </rect>
           </property>
           <property name="text">
            <string>0</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_pSleep">
           <property name="geometry">
            <rect>
             <x>70</x>
             <y>70</y>
             <width>67</width>
             <height>17</height>
            </rect>
           </property>
           <property name="text">
            <string>0</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_pZombie">
           <property name="geometry">
            <rect>
             <x>70</x>
             <y>100</y>
             <width>67</width>
             <height>17</height>
            </rect>
           </property>
           <property name="text">
            <string>0</string>
           </property>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="pushButton_reboot">
         <property name="minimumSize">
          <size>
           <width>89</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>89</width>
           <height>25</height>
          </size>
         </property>
         <property name="text">
          <string>重启</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_halt">
         <property name="minimumSize">
          <size>
           <width>89</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>89</width>
           <height>25</height>
          </size>
         </property>
         <property name="text">
          <string>关机</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
   <zorder>widget</zorder>
   <zorder>tabWidget</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>638</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
